WebContent/jsp/common/getAllModules.jsp

```diff
@@ -0,0 +1,35 @@
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+
+
+<div class="row">
+
+	<!-- this is bootstrap class "center the content"  ( col-md-offset-3 --will be different for each col-md-6 ) -->
+	
+	<div class="col-md-6 col-md-offset-3">

```

Comment:
this how we can make the div center align --using bootstarp
Refer:
https://stackoverflow.com/questions/18153234/center-a-column-using-twitter-bootstrap-3